### PR TITLE
[s] Fixes config reading exploit

### PR DESCRIPTION
**Pull request by Cyberboss**
> Create foolproof system to prevent VV reading configs

> No guards on the access procs

REALLY GRINDS THE GEARS

### CODE REVIEW

File: code/controllers/configuration/configuration.dm [hunk 1]
```diff
@@ -128,6 +131,9 @@ GLOBAL_PROTECT(config_dir)
 	return E.value
 
 /datum/controller/configuration/proc/Set(entry_type, new_val)
+	if(IsAdminAdvancedProcCall() && GLOB.LastAdminCalledProc == "Set" && GLOB.LastAdminCalledTargetRef == "\ref[src]")
+		log_admin_private("Config rewrite of [entry_type] to [new_val] attempted by [key_name(usr)]")
```
Comment (kevinz000) [inline code/controllers/configuration/configuration.dm]: 😂

**Comment by kevinz000** [comment]
nothing is foolproof

**Comment by Cyberboss** [comment]
s/fool/kevinz000 textbook power-user

**Comment by Cyberboss** [comment]
Self-speed-merging because security

**Pull request merged**